infrastructure/nginx/sites-available/production

```diff
@@ -0,0 +1,86 @@
+server {
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+	server_name _;
+
+	root /var/www/html;
+	index index.html index.htm;
+
+	access_log  /var/log/nginx/access.log combined;
+	error_log   /var/log/nginx/error.log  error;
+
+	keepalive_timeout 20s;
+	sendfile  on;
+	tcp_nopush  on;
+
+	# Compression
+	gzip on;
+	gzip_comp_level 5;
+	gzip_min_length 256;
+	gzip_proxied any;
+	gzip_vary  on;
+	gzip_types
+    application/atom+xml
+    application/javascript
+    application/json
+    application/ld+json
+    application/manifest+json
+    application/rss+xml
+    application/vnd.geo+json
+    application/vnd.ms-fontobject
+    application/x-font-ttf
+    application/x-web-app-manifest+json
+    application/xhtml+xml
+    application/xml
+    font/opentype
+    image/bmp
+    image/svg+xml
+    image/x-icon
+    text/cache-manifest
+    text/css
+    text/plain
+    text/vcard
+    text/vnd.rim.location.xloc
+    text/vtt
+    text/x-component
+		text/x-cross-domain-policy;
+
+	charset utf-8;
+	charset_types
+    text/css
+    text/plain
+    text/vnd.wap.wml
+    application/javascript
+    application/json
+    application/rss+xml
+		application/xml;
+
+	error_page 404 error_404.html
+
+	# Security
+	add_header X-XSS-Protection "1; mode=block" always;
+	add_header X-Content-Type-Options nosniff always;
+	add_header X-Frame-Options SAMEORIGIN always;
+
+	# Cross Domain API
+	add_header "Access-Control-Allow-Origin" "*";
```

Comment:
As above. At least in production I would enforce the same-origin policy (at least for `POST` and `PUT`).

Reply:
Yes you're right, it does not matter to be cross-origin in development. I will remove this line.

Reply:
Done 👍